src/core/types/arbitrageloops/skipLoop.ts

```diff
@@ -158,6 +199,9 @@ export class SkipLoop extends MempoolLoop {
 
 		if (res.result.code !== 0) {
 			logMessage += `\t **error code:** ${res.result.code}\n**error:** ${res.result.error}\n`;
+			if (res.result.code == 4) {
+				await this.trade(arbTrade);
+			}
 		}
```

Comment:
smort, i also did this on my local @nick134-bit

Reply:
tho i'd love a === explicit check

Reply:
and we probably should have an option to disable this, as not everyone is willing to risk the fees